src/src_cpp/exe/merge/merge_mps.cpp

```diff
@@ -174,9 +180,19 @@ int main(int argc, char** argv)
 		investCost_l += x0[pairNameId.first] * costCoeff_l;
 	}
 
+	double overallCost_l = mergedSolver_l.Objective().Value();
+	double operationalCost_l = overallCost_l - investCost_l;
+
+	std::ostringstream oss_l;
+	//print_solution(oss_l, x0, true);
+	std::cout << oss_l.str();
+	LOG(INFO) << oss_l.str() << std::endl;
```

Comment:
These lines do nothing, since `oss_l` is empty.

Reply:
It's a mistake. They were deleted in the commit 8fd2d3ddf9ac943bf8cfaad8e8bfa9e5ea699d08 and I added they back when I merged...